api/v1_storage.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+// DP API v1 -- Storage
+
+//===========================================================================
+// storage/clients/:clientid
+
+function api_v1_storage_clients(string $method, array $data, array $query_params)
+{
+    global $pguser;
+
+    $clientid = $data[":clientid"];
+
+    $storage = new ApiClientStorage($clientid, $pguser);
+    if ($method == "GET") {
+        return json_decode($storage->get());
```

Comment:
Should not the client decode and enode the JSON data so we are just dealing with strings here?

Reply:
The API automatically converts returns to JSON values so if we don't decode this into an object we stringify it twice.

Reply:
Yes, of course, you are right.

Reply:
But I'm still concerned about this. As I've discovered elsewhere, converting between javascript objects and arrays and php arrays and associative arrays via JSON can cause some screw-ups. Let me think about this some more.